signale.js

```diff
@@ -163,7 +167,7 @@ class Signale {
         [msg] = args;
       } else {
         const [{prefix, message, suffix}] = args;
-        msg = message ? util.format(...message) : util.format(...args);
+        msg = message ? util.format(...message) : this._prefixSufixNestedTernary(prefix, suffix, ...args);
         additional = Object.assign({}, {suffix, prefix});
```

Comment:
Just a minor reordering of the assignments:

```js
const [{prefix, message, suffix}] = args;
additional = Object.assign({}, {suffix, prefix});
msg = message ? util.format(...message) : this._hasAdditional(additional, args);
```

Reply:
It's better than my previous method